### PR TITLE
docs: Update docker quick-start guide and adds relevant project URLS

### PR DESCRIPTION
**Pull request by koolhead17**
## Description
This PR cleans docker quick-start guide and adds relevant project URLS in the ``Explore Further`` section. 
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/orchestration/docker-compose/README.md [hunk 1]
```diff
@@ -43,9 +43,6 @@ Each instance is now accessible on the host at ports 9001 through 9004, proceed
 * Minio services in the Docker compose file expose ports 9001 to 9004. This allows multiple services to run on a host.
 
 ### Explore Further
+- [Overview of Docker Compose](https://docs.docker.com/compose/overview/)
+- [Minio Docker Quickstart Guide](https://docs.minio.io/docs/minio-docker-quickstart-guide)
```
Comment (nitisht) [inline docs/orchestration/docker-compose/README.md:47]: Minio Docker Swarm doc link can be added here. https://docs.minio.io/docs/deploy-minio-on-docker-swarm

File: docs/orchestration/docker-swarm/README.md [hunk 2]
```diff
@@ -63,8 +63,5 @@ docker stack rm minio_stack
 
 ### Explore Further
 - [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
-- [Use `mc` with Minio Server](https://docs.minio.io/docs/minio-client-quickstart-guide)
-- [Use `aws-cli` with Minio Server](https://docs.minio.io/docs/aws-cli-with-minio)
-- [Use `s3cmd` with Minio Server](https://docs.minio.io/docs/s3cmd-with-minio)
-- [Use `minio-go` SDK with Minio Server](https://docs.minio.io/docs/golang-client-quickstart-guide)
-- [The Minio documentation website](https://docs.minio.io)
+- [Overview of Docker Swarm mode](https://docs.docker.com/engine/swarm/)
+- [Minio Docker Quickstart Guide](https://docs.minio.io/docs/minio-docker-quickstart-guide)
```
Comment (nitisht) [inline docs/orchestration/docker-swarm/README.md]: Minio Docker compose link can be added here, https://docs.minio.io/docs/deploy-minio-on-docker-compose

File: docs/orchestration/dcos/README.md [hunk 3]
```diff
@@ -36,8 +36,4 @@ $ dcos package uninstall minio
 
 ### Explore Further
 - [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
-- [Use `mc` with Minio Server](https://docs.minio.io/docs/minio-client-quickstart-guide)
-- [Use `aws-cli` with Minio Server](https://docs.minio.io/docs/aws-cli-with-minio)
-- [Use `s3cmd` with Minio Server](https://docs.minio.io/docs/s3cmd-with-minio)
-- [Use `minio-go` SDK with Minio Server](https://docs.minio.io/docs/golang-client-quickstart-guide)
-- [The Minio documentation website](https://docs.minio.io)
+- [DC/OS Project](https://dcos.io/)
```
Comment (nitisht) [inline docs/orchestration/dcos/README.md]: DC/OS documentation link can be added https://docs.mesosphere.com/ instead of dcos.io

**Review by nitisht — CHANGES_REQUESTED** [review]
LGTM except few small comments.

**Comment by harshavardhana** [comment]
What is supposed to be the PR title here ? What is 035 ?

**Review by harshavardhana — CHANGES_REQUESTED** [review]
PR title needs to be fixed first.  A PR title should concisely address what the PR is supposed to be before being submitted for review.

PR is also a way of documentation for future reference and understanding. When we observe something like this, the owner of the PR should quickly fix it as well.

**Review by nitisht — APPROVED** [review]

File: docs/orchestration/docker-swarm/README.md [hunk 4]
```diff
@@ -62,6 +62,9 @@ docker stack rm minio_stack
 * Docker Swarm uses ingress load balancing by default. You can configure [external load balancer based](https://docs.docker.com/engine/swarm/ingress/#/configure-an-external-load-balancer) on requirements.
 
 ### Explore Further
-- [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
 - [Overview of Docker Swarm mode](https://docs.docker.com/engine/swarm/)
 - [Minio Docker Quickstart Guide](https://docs.minio.io/docs/minio-docker-quickstart-guide)
+- [Deploy Minio on Docker Compose](https://docs.minio.io/docs/deploy-minio-on-docker-compose)
+- [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
+
```
Comment (harshavardhana) [inline docs/orchestration/docker-swarm/README.md]: Spurious space m

File: docs/orchestration/docker-swarm/README.md [hunk 5]
```diff
@@ -62,9 +62,9 @@ docker stack rm minio_stack
...
+- [Overview of Docker Swarm mode](https://docs.docker.com/engine/swarm/)
+- [Minio Docker Quickstart Guide](https://docs.minio.io/docs/minio-docker-quickstart-guide)
+- [Deploy Minio on Docker Compose](https://docs.minio.io/docs/deploy-minio-on-docker-compose)
 - [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
-- [Use `mc` with Minio Server](https://docs.minio.io/docs/minio-client-quickstart-guide)
-- [Use `aws-cli` with Minio Server](https://docs.minio.io/docs/aws-cli-with-minio)
-- [Use `s3cmd` with Minio Server](https://docs.minio.io/docs/s3cmd-with-minio)
-- [Use `minio-go` SDK with Minio Server](https://docs.minio.io/docs/golang-client-quickstart-guide)
-- [The Minio documentation website](https://docs.minio.io)
+
```
Comment (harshavardhana) [inline docs/orchestration/docker-swarm/README.md]: Spurious space

File: docs/docker/README.md [hunk 6]
```diff
@@ -113,6 +110,9 @@ docker stats <container_id>
 
 ## Explore Further
 
+* [Deploy Minio on Docker Compose](https://docs.minio.io/docs/deploy-minio-on-docker-compose)
+* [Deploy Minio on Docker Swarm](https://docs.minio.io/docs/deploy-minio-on-docker-swarm)
+* [Distributed Minio Quickstart Guide](https://docs.minio.io/docs/distributed-minio-quickstart-guide)
 * [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
-* [Distributed Minio Quickstart Guide ](https://docs.minio.io/docs/distributed-minio-quickstart-guide)
-* [Docker Compose](https://docs.docker.com/compose/)
+
```
Comment (harshavardhana) [inline docs/docker/README.md]: Spurious space.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**